File: JavaScript/Problem010.js
let problem10 = maxExclusive => {
    let IsPrime = number => {
        if (number <= 1) {return false;}
        if (number == 2) {return true;}
        if (number % 2 == 0) {return false;}

        let boundary = Math.floor(Math.sqrt(number));

        for (let i = 3; i <= boundary; i += 2) {
            if (number % i == 0) {return false;}
        }

        return true;
    };

    let value = 0;

    for(let i = 0; i <= maxExclusive; ++i) {
        if(IsPrime(i)) {
            value += i;
        }
    }

    return value;
}

File: Java/Problem010.java
public class Problem010 {
    private static boolean isPrime(long n) {
        if(n == 1) return false;
        if(n == 2) return true;

        for(long i = 2; i <= Math.sqrt(n); ++i) {
            if(n % i == 0) {
                return false;
            }
        }

        return true;
    }

    public static void main(String[] args) {
        long sum = 0;

        for(long i = 1; i < 2000000; ++i){
            if(isPrime(i)) {
                sum += i;
            }
        }

        System.out.println(sum);
    }
}